pkg/cmd/node/node.go

```diff
@@ -49,6 +49,18 @@ func diagnosticsClient() *diagnostics.Client {
 	return diagnostics.NewClient(pluginutil.HTTPClient(""))
 }
 
-func mesosClient() *mesos.Client {
+func mesosDNSClient() *mesos.Client {
```

Comment:
I don't think `mesosDNSClient` is the right name for this as it does not return a different client class. `mesosDefaultClient` would be clearer IMO.